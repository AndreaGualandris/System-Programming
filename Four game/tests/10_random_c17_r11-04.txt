1
2
12
3
13
6
13
13
3
14
14
11
5
3
10
5
6
12
7
0
1
9
13
11
5
4
3
13
14
15
15
2
2
16
8
9
13
5
16
13
16
12
15
9
1
1
15
16
15
16
16
0
0
4
16
16
16
14
11
12
14
8
0
0
1
16
3
6
3
3
5
0
0
4
0
0
5
4
3
6
0
16
16
6
5
11
14
6
15
16
5
12
5
15
13
1
6
4
15
12
1
16
4
3
12
14
10
2
8
11
12
14
8
0
5
0
15
10
6
6
4
11
16
0
11
6
16
9
3
4
14
6
7
8
16
15
11
0
8
16
12
15
11
6
16
14
1
8
1
7
6
7
5
8
3
1
13
2
5
3
13
6
8
7
15
1
6
16
1
0
13
3
2
15
9
5
16
3
2
7
2
13
9
15
9
5
14
