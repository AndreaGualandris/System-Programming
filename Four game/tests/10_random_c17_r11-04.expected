-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
-----------------
01234567890123456
Player 'o' to move:
 o
 v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
 o               
-----------------
01234567890123456
Player 'x' to move:
  x
  v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
 ox              
-----------------
01234567890123456
Player 'o' to move:
            o
            v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
 ox         o    
-----------------
01234567890123456
Player 'x' to move:
   x
   v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
 oxx        o    
-----------------
01234567890123456
Player 'o' to move:
             o
             v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
 oxx        oo   
-----------------
01234567890123456
Player 'x' to move:
      x
      v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
 oxx  x     oo   
-----------------
01234567890123456
Player 'o' to move:
             o
             v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
             o   
 oxx  x     oo   
-----------------
01234567890123456
Player 'x' to move:
             x
             v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
             x   
             o   
 oxx  x     oo   
-----------------
01234567890123456
Player 'o' to move:
   o
   v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
             x   
   o         o   
 oxx  x     oo   
-----------------
01234567890123456
Player 'x' to move:
              x
              v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
             x   
   o         o   
 oxx  x     oox  
-----------------
01234567890123456
Player 'o' to move:
              o
              v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
             x   
   o         oo  
 oxx  x     oox  
-----------------
01234567890123456
Player 'x' to move:
           x
           v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
             x   
   o         oo  
 oxx  x    xoox  
-----------------
01234567890123456
Player 'o' to move:
     o
     v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
             x   
   o         oo  
 oxx ox    xoox  
-----------------
01234567890123456
Player 'x' to move:
   x
   v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
   x         x   
   o         oo  
 oxx ox    xoox  
-----------------
01234567890123456
Player 'o' to move:
          o
          v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
   x         x   
   o         oo  
 oxx ox   oxoox  
-----------------
01234567890123456
Player 'x' to move:
     x
     v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
   x         x   
   o x       oo  
 oxx ox   oxoox  
-----------------
01234567890123456
Player 'o' to move:
      o
      v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
   x         x   
   o xo      oo  
 oxx ox   oxoox  
-----------------
01234567890123456
Player 'x' to move:
            x
            v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
   x         x   
   o xo     xoo  
 oxx ox   oxoox  
-----------------
01234567890123456
Player 'o' to move:
       o
       v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
   x         x   
   o xo     xoo  
 oxx oxo  oxoox  
-----------------
01234567890123456
Player 'x' to move:
x
v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
   x         x   
   o xo     xoo  
xoxx oxo  oxoox  
-----------------
01234567890123456
Player 'o' to move:
 o
 v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
   x         x   
 o o xo     xoo  
xoxx oxo  oxoox  
-----------------
01234567890123456
Player 'x' to move:
         x
         v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
   x         x   
 o o xo     xoo  
xoxx oxo xoxoox  
-----------------
01234567890123456
Player 'o' to move:
             o
             v
-----------------
                 
                 
                 
                 
                 
                 
                 
             o   
   x         x   
 o o xo     xoo  
xoxx oxo xoxoox  
-----------------
01234567890123456
Player 'x' to move:
           x
           v
-----------------
                 
                 
                 
                 
                 
                 
                 
             o   
   x         x   
 o o xo    xxoo  
xoxx oxo xoxoox  
-----------------
01234567890123456
Player 'o' to move:
     o
     v
-----------------
                 
                 
                 
                 
                 
                 
                 
             o   
   x o       x   
 o o xo    xxoo  
xoxx oxo xoxoox  
-----------------
01234567890123456
Player 'x' to move:
    x
    v
-----------------
                 
                 
                 
                 
                 
                 
                 
             o   
   x o       x   
 o o xo    xxoo  
xoxxxoxo xoxoox  
-----------------
01234567890123456
Player 'o' to move:
   o
   v
-----------------
                 
                 
                 
                 
                 
                 
                 
   o         o   
   x o       x   
 o o xo    xxoo  
xoxxxoxo xoxoox  
-----------------
01234567890123456
Player 'x' to move:
             x
             v
-----------------
                 
                 
                 
                 
                 
                 
             x   
   o         o   
   x o       x   
 o o xo    xxoo  
xoxxxoxo xoxoox  
-----------------
01234567890123456
Player 'o' to move:
              o
              v
-----------------
                 
                 
                 
                 
                 
                 
             x   
   o         o   
   x o       xo  
 o o xo    xxoo  
xoxxxoxo xoxoox  
-----------------
01234567890123456
Player 'x' to move:
               x
               v
-----------------
                 
                 
                 
                 
                 
                 
             x   
   o         o   
   x o       xo  
 o o xo    xxoo  
xoxxxoxo xoxooxx 
-----------------
01234567890123456
Player 'o' to move:
               o
               v
-----------------
                 
                 
                 
                 
                 
                 
             x   
   o         o   
   x o       xo  
 o o xo    xxooo 
xoxxxoxo xoxooxx 
-----------------
01234567890123456
Player 'x' to move:
  x
  v
-----------------
                 
                 
                 
                 
                 
                 
             x   
   o         o   
   x o       xo  
 oxo xo    xxooo 
xoxxxoxo xoxooxx 
-----------------
01234567890123456
Player 'o' to move:
  o
  v
-----------------
                 
                 
                 
                 
                 
                 
             x   
   o         o   
  ox o       xo  
 oxo xo    xxooo 
xoxxxoxo xoxooxx 
-----------------
01234567890123456
Player 'x' to move:
                x
                v
-----------------
                 
                 
                 
                 
                 
                 
             x   
   o         o   
  ox o       xo  
 oxo xo    xxooo 
xoxxxoxo xoxooxxx
-----------------
01234567890123456
Player 'o' to move:
        o
        v
-----------------
                 
                 
                 
                 
                 
                 
             x   
   o         o   
  ox o       xo  
 oxo xo    xxooo 
xoxxxoxooxoxooxxx
-----------------
01234567890123456
Player 'x' to move:
         x
         v
-----------------
                 
                 
                 
                 
                 
                 
             x   
   o         o   
  ox o       xo  
 oxo xo  x xxooo 
xoxxxoxooxoxooxxx
-----------------
01234567890123456
Player 'o' to move:
             o
             v
-----------------
                 
                 
                 
                 
                 
             o   
             x   
   o         o   
  ox o       xo  
 oxo xo  x xxooo 
xoxxxoxooxoxooxxx
-----------------
01234567890123456
Player 'x' to move:
     x
     v
-----------------
                 
                 
                 
                 
                 
             o   
             x   
   o x       o   
  ox o       xo  
 oxo xo  x xxooo 
xoxxxoxooxoxooxxx
-----------------
01234567890123456
Player 'o' to move:
                o
                v
-----------------
                 
                 
                 
                 
                 
             o   
             x   
   o x       o   
  ox o       xo  
 oxo xo  x xxoooo
xoxxxoxooxoxooxxx
-----------------
01234567890123456
Player 'o' wins!
